### PR TITLE
Add new coach report workflows

### PR DESCRIPTION
**Pull request by jonboiser**
### Summary

1. This adds some new workflows to coach reports and modifies some existing ones
1. Deletes ununsed codes and tries to replace some translation strings with strings in the commonCoach module.

**New: Preview and edit quiz details from within a quiz report page**

Screen Shot 2019-04-17 at 12 32 31 PM

**New: Edit lesson details and manage resources from within a lesson report page**

Screen Shot 2019-04-17 at 12 32 21 PM

**New: View a quiz preview from the Plan Page**

Screen Shot 2019-04-17 at 12 32 09 PM

**Changed: Editing Lessons/Quizzes happens in its own page**
Screen Shot 2019-04-17 at 12 36 31 PM

…

### Reviewer guidance

1. Manually test each of the new workflows
1. Pay special attention to canceling out of workflows

### References

…

----

### Contributor Checklist

PR process:

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [ ] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [x] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

Testing:

- [ ] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

### Reviewer Checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

**Comment by rtibbles** [comment]
Looks like you need to relint the frontend after the merge to comply with @MisRob's latest linting rules!

File: kolibri/core/assets/src/views/KTextbox/KeenUiTextbox.vue [hunk 1]
```diff
@@ -61,8 +61,8 @@
           v-else
           ref="textarea"
 
-          v-model="value"
```
Comment (rtibbles) [inline kolibri/core/assets/src/views/KTextbox/KeenUiTextbox.vue:64]: Odd - wonder why KeenUI had this as a v-model, when the input above is v-bind?

**Review by rtibbles — APPROVED** [review]
Manual testing checks out, nothing jumps out of the code at me.

**Comment by indirectlylit** [comment]
updated process labels

image

and created new issues for the stories and docs:

* https://github.com/learningequality/kolibri/issues/5399
* https://github.com/learningequality/kolibri-docs/issues/82

**Pull request merged**